src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,178 @@
+        try {
+            File myObj = new File(filePath);
+            if (myObj.createNewFile()) {
+                System.out.println("No tasks at the moment");
+            } else {
+                try {
+                    printFileContents(filePath);
+                } catch (FileNotFoundException e) {
+                    System.out.println("File not found");
+                }
+            }
+        } catch (IOException e) {
+            System.out.println("An error occurred.");
+            e.printStackTrace();
+        }
+        return storeLists;
+    }
+
+    /**
+     * Prints File Contents onto console
+     * @param filePath Location where file is stored
+     * @throws FileNotFoundException Throws exception if file is not available
+     */
+    public void printFileContents(String filePath) throws FileNotFoundException {
+        try (BufferedReader br = new BufferedReader(new FileReader(filePath))) {
+            String line;
+            while ((line = br.readLine()) != null) {
+                renderStringsAsTasks(line);
+            }
+        } catch (IOException ex) {
+            ex.printStackTrace();
+        }
+    }
+
+    /**
+     * Method that parses strings and converts them to Tasks
+     * @param taskStr String that will be passed into method to convert
+     */
+    public void renderStringsAsTasks(String taskStr) {
+        String taskType = String.valueOf(taskStr.charAt(1));
+        String taskDescription = taskStr.split("] ", 2)[1];
+        if (taskType.equals("T")) {
+            Todo t = new Todo(taskDescription);
+            storeLists.add(t);
+        } else if (taskType.equals("D")) {
+            String duration = taskDescription.split("y: ", 2)[1];
+            taskDescription = taskDescription.split("\\(", 2)[0];
+            Deadline d = new Deadline(taskDescription, duration);
+            storeLists.add(d);
+        } else {
+            String duration = taskDescription.split("t: ", 2)[1];
+            taskDescription = taskDescription.split("\\(", 2)[0];
+            Event e = new Event(taskDescription, duration);
+            storeLists.add(e);
+        }
+    }
+
+    /**
+     * To add to file when tasks are added
+     * @param textToAppend Text that is required to be added
+     */
+    public void appendToFile(String textToAppend) {
+        try {
+            File newFile = new File(this.filePath);
+            FileWriter fw = new FileWriter(this.filePath, true); // create a FileWriter in append mode
+            if(newFile.length() == 0) {
+                fw.write(textToAppend);
+            } else {
+                fw.write("\n" + textToAppend);
+            }
+            fw.close();
+        } catch (IOException ex) {
+            ex.printStackTrace();
+        }
+    }
+
+    /**
+     * To edit Text in file primarily to mark or unmark tasks
+     * @param newText New text to replace old text
+     * @param oldText Old text that is meant to be replaced
```

Comment:
Should theses sentences be terminated with a full stop `.`? 
These minor nits occur in several of the javadocs